Enferno.Services.StormConnect.Contracts/Common/Image.cs

```diff
@@ -8,13 +9,26 @@ public class Image
     {
         [DataMember(Order = 1)]
         public string FileTypeName { get; set; }
+
         [DataMember(Order = 2)]
         public string ExternalCode { get; set; }
+
         [DataMember(Order = 3)]
         public string Filename { get; set; }
+
         [DataMember(Order = 4)]
         public string Url { get; set; }
+
         [DataMember(Order = 5)]
         public DateTime? LastModified { get; set; }
+
+        [DataMember(Order = 6)]
+        public List<ImageCulture> Cultures { get; set; }
+
+        [DataMember(Order = 7)]
+        public bool? IsVariantUnique { get; set; }
```

Comment:
This is not needed. A list of Image objects are found both on Product.AdditionalImages and Product.Variants[n].AdditionalImages. Those found on Product sets IsVariantUnique to false and those on Variant sets it to true. This is already handled in  ProductImageProcessor.EnsureDbProductFile which takes a variable depending on if it is found on Product or Variant.